src/main/webapp/itempagescript.js

```diff
@@ -133,7 +133,7 @@ function getItemPageComments(comments) {
         "(" +
         date.toLocaleString() +
         ")", commentDataManager.belongsToUser
-      )
+      ,commentDataManager.currentEmail)
```

Comment:
Add a space after the comma:
```suggestion
      , commentDataManager.currentEmail)
```